releasenotes/notes/transpile-617bd3a4e6f1c0d8.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Adds `BaseExperiment._transpile` which is called to transpile the
+    experiment circuits before sending them to the backend. It can be
+    overridden to define custom transpilation.
+deprecations:
+  - |
+    `BaseExperiment._postprocess_transpiled_circuits` is deprecated and will
+    be removed in the 0.4.0 release. Use `BaseExperiment._transpile` instead.
```

Comment:
This comment also needs updating to refer to the correct method. You should use RST links as well eg
```
:meth:`.BaseExperiment._transpiled_circuits`
```

Reply:
Done.